cmd/egress/cmd.go

```diff
@@ -26,22 +26,24 @@ var (
 )
 
 type egressConfig struct {
-	vpcSubnetID     string
-	cloudImageID    string
-	instanceType    string
-	securityGroupId string
-	cloudTags       map[string]string
-	debug           bool
-	region          string
-	timeout         time.Duration
-	kmsKeyID        string
-	httpProxy       string
-	httpsProxy      string
-	CaCert          string
-	noTls           bool
-	gcp             bool
-	awsProfile      string
-	gcpVpcName      string
+	vpcSubnetID                string
+	cloudImageID               string
+	instanceType               string
+	securityGroupId            string
+	cloudTags                  map[string]string
+	debug                      bool
+	region                     string
+	timeout                    time.Duration
+	kmsKeyID                   string
+	httpProxy                  string
+	httpsProxy                 string
+	CaCert                     string
+	noTls                      bool
+	gcp                        bool
+	awsProfile                 string
+	gcpVpcName                 string
+	SkipAWSInstanceTermination bool
+	TerminateDebugInstance     string
```

Comment:
```suggestion
	skipAWSInstanceTermination bool
	terminateDebugInstance     string
```
We may want to make these private configuration options just like the rest